src/js/angular/rest/export.rest.service.js

```diff
@@ -0,0 +1,50 @@
+angular
+    .module('graphdb.framework.rest.export.service', [])
+    .factory('ExportRestService', ExportRestService);
+
+ExportRestService.$inject = ['$http', '$repositories', '$translate'];
+
+const REPOSITORIES_ENDPOINT = 'repositories';
+function ExportRestService($http, $repositories, $translate) {
+    return {
+        getExportedStatementsAsJSONLD
+    };
+
+    function getExportedStatementsAsJSONLD(context, repo, graphsByValue, acceptHeader, linkHeader, auth, forSelectedGraphs) {
+        let url;
+        if (forSelectedGraphs) {
+            url = `${REPOSITORIES_ENDPOINT}/${repo.id}/statements?infer=false&${context}`;
+        } else {
+            if (context) {
+                url = `${REPOSITORIES_ENDPOINT}/${repo.id}/statements?infer=false&context=${graphsByValue[context.value].exportUri}&location=${encodeURIComponent(repo.location)}`;
+            } else {
+                url = `${REPOSITORIES_ENDPOINT}/${repo.id}/statements?infer=false&location=${encodeURIComponent(repo.location)}`;
+            }
+        }
```

Comment:
There is quite a lot of repetition here. Can you try to simplify this by extracting the url base which is repeated in all branches and then just add on top of it the other parameters.